frontend/www/js/omegaup/arena/arena.js

```diff
@@ -279,12 +281,35 @@ export class Arena {
       clarification: $('#clarification'),
       clock: $('#title .clock'),
       loadingOverlay: $('#loading'),
-      miniRanking: $('#mini-ranking'),
-      problemList: $('#problem-list'),
       ranking: $('#ranking div'),
       socketStatus: $('#title .socket-status'),
       submitForm: $('#submit'),
     };
+
+    const problemListPayload = JSON.parse(
+      document.getElementById('problem-list-payload').innerText,
+    );
+    self.elements.navBar = new Vue({
+      el: '#arena-navbar',
+      render: function(createElement) {
+        return createElement('omegaup-arena-navbar', {
+          props: {
+            showRanking: this.showRanking,
+            problemsList: this.problemsList,
```

Comment:
```suggestion
            problems: this.problems,
```

no es necesario decir que es una lista porque tienes el plural para ayudar con eso.

Reply:
Listo